### DISCUSSION TITLE
Unable to process the #XXX link in Readme properly

### DISCUSSION THREAD

**Discussion by SheepChef** [community/community#147026] [Mobile]
The Mobile version of GitHub **cannot** jump to a specific section of README via a link like:

```markdown

[Text](#title)

```

However everything works fine in the browser, and I have checked that my markdown link is in the correct format.

All in all, I consider that this is a bug, so I am reporting it here.

**Comment by SheepChef** [comment]
I am not showing any reproduction method, since it is very easy to reproduce. Just create a Readme file with some titles and a link refers to one of them.

**Comment by SheepChef** [comment]
Any update?

**Comment by iskae** [comment]
Hi @SheepChef 👋 

Sorry for not responding sooner. I _assume_ you are using an Android device, right? I gave this a shot just now on iOS and I was not able to reproduce it there, but it is easily reproducible on Android 😅 

Creating a bug issue for us and will be working on it asap 🙇 Thanks so much for reporting this ❤️ 

Cheers!

**Reply by SheepChef** [reply]
I fired a bug ticket via the GitHub support site.
